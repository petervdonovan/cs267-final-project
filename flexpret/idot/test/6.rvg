[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [single-else-core t0
      [single-hart t3 [sim-print-int 42 t1 t2]]
      [single-hart t3 [sim-print-int 99 t1 t2]]]
    [id die]}]]
]
