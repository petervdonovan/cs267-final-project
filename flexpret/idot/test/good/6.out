Core-0:         42
Core-1:         99
Core-2:         99
Core-3:         99
Core-4:         99
Core-5:         99
Core-6:         99
Core-7:         99
Core-8:         99
Core-9:         99
Core-10:         99
Core-11:         99
Core-12:         99
Core-13:         99
Core-14:         99
Core-15:         99
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
